SwaprHQ/swapr-dapp
# [SWA-30][FEAT] - Add $ARB token to common tokens

# ⚠️ Waiting for SDK updates to be released ⚠️ 

## Fixes:
* #1814 
* [SWA-30](https://linear.app/swaprdev/issue/SWA-30/add-arb-token-to-common-tokens)

# Description

* Add $ARB to common tokens suggestions for Ethereum Mainnet, Arbitrum One, and Arbitrum Goerli

![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/13aa9885-7cbc-4818-9588-79fda71f0ed5)
![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/1d2e2c05-dbb8-477c-bf66-e3875063899c)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet
5) $ARB token should be among the common tokens suggested for swaps on Ethereum Mainnet, Arbitrum One and Arbitrum Goerli
